### ISSUE TITLE
allow "latest" for .ruby-version and/or make "latest" the default behavior

### ISSUE THREAD

**Issue by cauerego** [netlify/build-image#208]
so, if there's no `.ruby-version`, it would be using the latest available (currently, 2.4.3 if i'm not mistaken).

not sure why this is not the default.

cheers! :)

**Comment by bcomnes** [comment]
Yes agreed.  We should add this.  In the meantime I am updating ruby versions to the latest.
